### ISSUE TITLE
cannot read property addchunk

### ISSUE THREAD

**Issue by carvinlo** [OpenNTF/dojo-webpack-plugin#115]
Hi, Thanks for your great job.
but i use html-webpack-plugin with dojo-webpack-plugin, something wrong:
“cannot read property addchunk”

**Comment by chuckdumont** [comment]
Please provide a stack trace.  If possible, a test case that reproduces the problem would be helpful.

**Comment by carvinlo** [comment]
I'm using the sample application at [https://github.com/OpenNTF/dojo-webpack-plugin-sample](https://github.com/OpenNTF/dojo-webpack-plugin-sample).

1. `npm i --save-dev html-webpack-plugin`
2. add new code in `webpack-config.js`
```js
var HtmlWebpackPlugin = require('html-webpack-plugin');
module.exports = {
    plugins: [
        new HtmlWebpackPlugin(),
```
3. run `npm run build`

the stack trace:
```js
Process finished normally.
        errors: 0
        warnings: 2
        build time: 0.081 seconds

Unhandled rejection TypeError: Cannot read property 'addChunk' of null
    at self.preparedChunks.forEach.preparedChunk (C:\dojo-webpack-plugin-sample-master\node_modules\webpack\lib\Compilation.js:586:11)
    at Array.forEach (<anonymous>)
    at Compilation.seal (C:\dojo-webpack-plugin-sample-master\node_modules\webpack\lib\Compilation.js:579:23)
    at applyPluginsParallel.err (C:\dojo-webpack-plugin-sample-master\node_modules\webpack\lib\Compiler.js:504:17)
    at C:\dojo-webpack-plugin-sample-master\node_modules\tapable\lib\Tapable.js:289:11
    at C:\dojo-webpack-plugin-sample-master\node_modules\html-webpack-plugin\index.js:60:9
    at tryCatcher (C:\dojo-webpack-plugin-sample-master\node_modules\bluebird\js\release\util.js:16:23)
    at Promise._settlePromiseFromHandler (C:\dojo-webpack-plugin-sample-master\node_modules\bluebird\js\release\promise.js:512:31)
    at Promise._settlePromise (C:\dojo-webpack-plugin-sample-master\node_modules\bluebird\js\release\promise.js:569:18)
    at Promise._settlePromise0 (C:\dojo-webpack-plugin-sample-master\node_modules\bluebird\js\release\promise.js:614:10)
    at Promise._settlePromises (C:\dojo-webpack-plugin-sample-master\node_modules\bluebird\js\release\promise.js:693:18)
    at Async._drainQueue (C:\dojo-webpack-plugin-sample-master\node_modules\bluebird\js\release\async.js:133:16)
    at Async._drainQueues (C:\dojo-webpack-plugin-sample-master\node_modules\bluebird\js\release\async.js:143:10)
    at Immediate.Async.drainQueues (C:\dojo-webpack-plugin-sample-master\node_modules\bluebird\js\release\async.js:17:14)
    at runCallback (timers.js:789:20)
    at tryOnImmediate (timers.js:751:5)
    at processImmediate [as _immediateCallback] (timers.js:722:5)
```
part of package.json
```js
  "devDependencies": {
    "copy-webpack-plugin": "4.1.1",
    "css-loader": "0.25.0",
    "dijit": "1.13.0",
    "dojo": "1.13.0",
    "dojo-util": "1.13.0",
    "dojo-webpack-plugin": "2.3.3",
    "dojox": "1.12.2",
    "less": "2.7.1",
    "less-loader": "2.2.3",
    "style-loader": "0.13.1",
    "url-loader": "0.5.7",
    "webpack": "3.10.0",
    "html-webpack-plugin": "^2.30.1"
  }
```

**Comment by chuckdumont** [comment]
Fixed in version 2.3.4
